lib/pure/parsecsv.nim

```diff
@@ -170,6 +170,9 @@ proc readRow*(my: var CsvParser, columns = 0): bool =
   ## Blank lines are skipped.
   var col = 0 # current column
   let oldpos = my.bufpos
+  while my.buf[my.bufpos] == '\c' or my.buf[my.bufpos] == '\l':
```

Comment:
I think `while .. in { .. }` would look nicer.

Reply:
Done